resources/template.html

```diff
@@ -219,7 +219,7 @@ <h4 class="panel-title">
     <footer>
       <p>
         Code: <a href="https://github.com/prometheus/pushgateway">prometheus/pushgateway</a>
-        Contact: <a href="mailto:prometheus-developers@googlegroups.com">prometheus-developers</a>
```

Comment:
What about just linking to our community page, so that we only have to care about one canonical page?